buildSrc/build.gradle.kts

```diff
@@ -148,7 +152,34 @@ if (!isCiServer || System.getProperty("enableCodeQuality")?.toLowerCase() == "tr
     apply { from("../gradle/codeQuality.gradle") }
 }
 
-apply { from("../gradle/ciReporting.gradle") }
+if (isCiServer) {
+    gradle.buildFinished {
+        tasks.all {
+            if (this is Reporting<*> && state.failure != null) {
+                val reportContainer = this.reports
+                val reportDestination = reportContainer.getByName("html").destination
+                prepareReportForCIPublishing(reportDestination)
```

Comment:
:x: These 3 lines can be simplified to `prepareReportForCIPublishing(this.reports["html"].destination)`

Reply:
Does not work for me: 
![image](https://user-images.githubusercontent.com/423186/36741806-55e26878-1be6-11e8-9c70-bc0d2a110f8c.png)

Do I need to import something?

Reply:
IntelliJ should suggest you to import from `org.gradle.kotlin.dsl.*`.